Examples/Filtering/DiffusionTensor3DReconstructionImageFilter.cxx

```diff
@@ -251,13 +251,19 @@ main(int argc, char * argv[])
           0.0) // this is a reference image
       {
         std::string fn("ReferenceImage%d.mhd");
-        snprintf(filename, sizeof(filename), fn.c_str(), referenceImageIndex);
+        ITK_GCC_PRAGMA_PUSH
+        ITK_GCC_SUPPRESS_Wformat_nonliteral snprintf(
+          filename, sizeof(filename), fn.c_str(), referenceImageIndex);
+        ITK_GCC_PRAGMA_POP
         ++referenceImageIndex;
       }
       else
       {
         std::string fn("Gradient%d.mhd");
-        snprintf(filename, sizeof(filename), fn.c_str(), i);
+        ITK_GCC_PRAGMA_PUSH
+        ITK_GCC_SUPPRESS_Wformat_nonliteral snprintf(
+          filename, sizeof(filename), fn.c_str(), i);
+        ITK_GCC_PRAGMA_POP
```

Comment:
Here as well, I think the variable `fn` may be removed, and the string literal `"Gradient%d.mhd"` may be passed directly to `snprintf`. No need to suppress the warning, I think!

Reply:
Thanks for the review @N-Dekker. Changed in 8984c2f.